docs/docs/providers/idme.md

```diff
@@ -0,0 +1,48 @@
+---
+id: idme
+title: ID.me
+---
+
+## Documentation
+
+https://developers.id.me/documentation
+
+## Configuration
+
+https://developers.id.me/organizations
+
+## Options
+
+The **ID.me Provider** comes with a set of default options:
+
+- [ID.me Provider options](https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/idme.ts)
+
+You can override any of the options to suit your own use case.
+
+## Example
+
+```js
+import IDMeProvider from "next-auth/providers/idme";
+...
+providers: [
+  IDMeProvider({
+    clientId: process.env.IDME_ID,
+    clientSecret: process.env.IDME_SECRET,
+    authorization: {
+      params: {
+        scope: 'openid military teacher',
+        redirect_uri: `${process.env.NEXTAUTH_URL}/api/auth/callback/idme`
+      }
+    }
+  })
+]
+...
+```
+
+:::warning
+The scope and redirect_uri fields are mandatory. Without them the authentication flow will fail. Do not forget to include openid as part of the scope.
+:::
```

Comment:
```suggestion
```

This is incorrect. NextAuth.js should include both by default.

Reply:
Will remove the redirect_uri piece.  However, setting the scope for ID.me does not include the openid in the url params.

If i do this:

```
    IDMeProvider({
      clientId: process.env.ID_ME_CLIENT_ID!,
      clientSecret: process.env.ID_ME_CLIENT_SECRET!,
      authorization: {
        params: {
          scope: 'military',
        },
      },
    }),
```

The url created is ```scope=military``` and results in a failure ```Try signing in with a different account```.  Even, removing the authorization above and letting the default provider settings inject ```scope=openid``` will result in the same error.

Let me do some checking because ID.me has two different verification options. The above was written for group verification which means that both openid and group (military, teacher, etc.) must be included in the url.

The identity platform (different api) may not need to have the additional scope defined in the provider instance.

Reply:
I meant, by default. IMO, this config should just be:

```ts
IDMeProvider({
  clientId: process.env.ID_ME_CLIENT_ID,
  clientSecret: process.env.ID_ME_CLIENT_SECRET,
}),
```

Overriding config options should already be explained in the "Options" section.